### DISCUSSION TITLE
Wording in Documentation. Does the documentation differ between Purchase and Subscription in its wording?

### DISCUSSION THREAD

**Discussion by Stophface** [hyochan/react-native-iap#2416] [Q&A]
Well, this is not a bug report. But I am wondering if the documentation distinguishes between the two very different processes of a *purchase*  and a *subscription*. Especially, here: https://react-native-iap.dooboolab.com/docs/guides/purchases

The example code differs between a purchase and a subscription

```
/**
   * For one-time products
   */
  render() {
      ...
  }

  /**
   * For subscriptions products
   */
  render() {
      ...
  }
```

Then, in the text it says: 

> Purchases
> Before you request any purchase, you should set purchaseUpdatedListener from react-native-iap. It is recommended that you start listening to updates as soon as your application launches. And don't forget that even at launch you may receive successful purchases that either completed while your app was closed or that failed to be finished, consumed or acknowledged due to network errors or bugs.

and 

> Finishing a Purchase
> Purchases will keep being emitted to your purchaseUpdatedListener on every app relaunch until you finish the purchase.
> All purchases should be consumed by calling finishTransaction().

I wonder if this is true only for purchases, or for subscriptions as well.

**Comment by andresesfm** [comment]
I think so, it also applies to subscriptions.
